.circleci/config.yml

```diff
@@ -343,6 +343,9 @@ jobs:
       - run:
           name: cargo clippy
           command: cargo xclippy --workspace --all-targets
+      - run:
+          name: cargo clippy tcb
+          command: cargo xclippy -p diem-secure-json-rpc -p diem-key-manager -p diem-secure-net -p diem-secure-push-metrics -p diem-secure-storage -p diem-secure-time -p safety-rules  -p execution-correctness
```

Comment:
I think this would be clearer:

```
command: |
  cargo xclippy \
    -p diem-secure-json-rpc \
    -p diem-key-manager \
...
```

also wondering:

1) are there any lints that we can enable today? 
2) looking at @jnaulty 's diagram, it looks like none of these are land-blocking (only the bors github-action are)

Reply:
CircleCi is always land blocking.